### PR TITLE
Fix/modules appearance

### PR DESCRIPTION
**Pull request by Crejak**
This PR focuses on visual enhancements of the app to fix issues and improve ergonomy.

## Changes

- Most major modules now use Window as their view (consultdoc, contribute, guided tours)
- Base demo now includes geocoding
- Changed the appearance of Base Demo to get rid of the wide left menu (now it is a fixed-size menu with icons, names are displayed on hover). The html is also cleaner (the "content view" node has the same size as we see it, got rid of unused CSS).
- Windows default location has been updated to correspond to the [interface concept](https://github.com/MEPP-team/RICT/blob/master/Doc/Devel/Design/Interface%20Design.md#web-demo-interface)

## Issues fixed

- #68, #71, #76 -> fixed by using Window for the view of main modules, changing their default position and updating the base demo interface (for #76, still have to manage the case where no guided tour is found)
- #70 : image src was incorrect
- #75 added popups
- #77 fixed opacity
- #78 updated index.html

## Screenshot : new interface

Screenshot_20190614_112718

### CODE REVIEW

**Comment by jailln** [comment]
Could you rebase on master and resolve the conflicts that appeared due to the merging of PR #87 please ?

**Comment by Crejak** [comment]
Rebase done.

File: UDV-Core/examples/DemoFull/Demo.js [hunk 1]
```diff
@@ -29,11 +29,13 @@ baseDemo.loadConfigFile('../data/config/generalDemoConfig.json').then(() => {
     const documents = new udvcore.DocumentController(baseDemo.view,
         baseDemo.controls, {temporal: baseDemo.temporal, active: false},
         baseDemo.config);
-    baseDemo.addModuleView('documents', documents);
+    baseDemo.addModuleView('documents', documents, {
+        binding: 'd'
+    });
```
[the remark this answers is not in the document]
Comment (Crejak) [reply UDV-Core/examples/DemoFull/Demo.js]: Exactly, it is explained in the jsdoc comment for `addModuleView`

File: UDV-Core/src/Extensions/Contribute/src/ContributeController.js [hunk 2]
```diff
@@ -122,6 +125,7 @@ export function ContributeController(documentController, requestService){
     for (var pair of this.visuData.entries()){
       length +=1;
     }
+    console.log(`${length} vs ${this.numberVisuData}`)
```
[the remark this answers is not in the document]
Comment (Crejak) [reply UDV-Core/src/Extensions/Contribute/src/ContributeController.js]: Yes, I removed it.

File: UDV-Core/src/Modules/ConsultDoc/DocumentBrowser.js [hunk 3]
```diff
@@ -276,13 +267,18 @@ export function DocumentBrowser(browserContainer, documentController) {
...
     };
 
-    this.startBrowser = function startBrowser() {
-        this.documentController.getDocuments();
+    startBrowser() {
+        try {
+            this.documentController.getDocuments();
+        } catch (e) {
+            //view is not created ?
+            return;
```
[the remark this answers is not in the document]
Comment (Crejak) [reply UDV-Core/src/Modules/ConsultDoc/DocumentBrowser.js]: Done

File: UDV-Core/src/Modules/GuidedTour/GuidedTour.js [hunk 4]
```diff
@@ -121,27 +132,25 @@ export function GuidedTour(guidedTourContainer, guidedTourController) {
...
       this.guidedTourController.toggleGuidedTourButtons(false);
-      document.getElementById("guidedTourText2").style.display = "inline-block";
       document.getElementById("guidedTourDocPreviewImg").style.display = "none";
       document.getElementById("guidedTourText1").style.height = "60%";
-      document.getElementById('docBrowserWindow').style.display = "block";
+      /*
       document.getElementById('docBrowserPreviousButton').style.display = "none";
       document.getElementById('docBrowserNextButton').style.display = "none";
-      document.getElementById('docBrowserIndex').style.display = "none";
+      document.getElementById('docBrowserIndex').style.display = "none";*/
```
[the remark this answers is not in the document]
Comment (Crejak) [reply UDV-Core/src/Modules/GuidedTour/GuidedTour.js]: Removed

File: UDV-Core/src/Modules/GuidedTour/GuidedTour.js [hunk 5]
```diff
@@ -152,20 +161,21 @@ export function GuidedTour(guidedTourContainer, guidedTourController) {
...
-  this.exitGuidedTour = function exitGuidedTour(){
+  exitGuidedTour(){
 
       this.guidedTourController.reset();
       // show the regular buttons for doc window
+      /*
       document.getElementById('docBrowserPreviousButton').style.display = "block";
       document.getElementById('docBrowserNextButton').style.display = "block";
-      document.getElementById('docBrowserIndex').style.display = "block";
+      document.getElementById('docBrowserIndex').style.display = "block";*/
```
[the remark this answers is not in the document]
Comment (Crejak) [reply UDV-Core/src/Modules/GuidedTour/GuidedTour.js]: Removed

**Pull request merged**